include/psp2/libime.h

```diff
@@ -0,0 +1,185 @@
+/**
+ * \usergroup{SceIme}
+ * \usage{psp2/libime.h,SceIme_stub,SCE_SYSMODULE_IME}

```

Comment:
how about just `ime.h`?

Reply:
nvm i forgot we already have `libdbg.h` and `libssl.h`

but i just thought `ime_dialog.h` & `libime.h` are pretty related in here.
(similarly, `common_dialog.h`  and `ime_dialog.h` are also related)
so i pass the decision role to other maintainers and @SonicMastr

Reply:
I also tried checking to see if any other header had the `lib` prefix before settling with `libime.h`. I stuck with it because I thought it would be a bit more accurate. It wouldn't be an issue to change it to `ime.h` if you want, though.

Reply:
it's not bad